pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -113,6 +126,7 @@ func (db *DB) Migration() *migrate.Migration {
 						`ALTER TABLE bandwidth_agreements ADD COLUMN total INT(10)`,
 						`ALTER TABLE bandwidth_agreements ADD COLUMN max_size INT(10)`,
 						`ALTER TABLE bandwidth_agreements ADD COLUMN created_utc_sec INT(10)`,
+						`ALTER TABLE bandwidth_agreements ADD COLUMN status INT(10) DEFAULT 0`,
```

Comment:
This DB migration will not be executed cause all storage nodes are running DB version 1 already. We need to move this to the next release which would be version 2.

Reply:
@aligeti You need to add next migration instead of modifying an existing one.